rust/main/config/mainnet_config.json

```diff
@@ -34,7 +34,7 @@
       "interchainAccountIsm": "0xd766e7C7517f2d0D92754b2fe4aE7AdEf7bDEC3e",
```

Comment:
I assume this is all drive-by due to some recent registry updates with the latest governance actions

Reply:
yup that's right